ports/esp8266/mpconfigport.h

```diff
@@ -198,4 +199,7 @@ extern const struct _mp_obj_module_t mp_module_onewire;
 #define MICROPY_WRAP_MP_KEYBOARD_INTERRUPT(f) MP_FASTCODE(f)
 #define MICROPY_WRAP_MP_SCHED_SCHEDULE(f) MP_FASTCODE(f)
 
+#define WDEV_HWRNG ((volatile uint32_t *)0x3ff20e44)
```

Comment:
This doesn't look right.  This value is [probably](https://github.com/esp8266/Arduino/blob/master/cores/esp8266/esp8266_peri.h#L848) [suitable](https://github.com/esp8266/Arduino/blob/master/cores/esp8266/esp8266_peri.h#L28) for the ESP8266 (I didn't find this in a [primary source](https://www.espressif.com/sites/default/files/documentation/esp8266-technical_reference_en.pdf).), but should be `0x3FF75144` per the [ESP32 Technical Reference Manual](https://www.espressif.com/sites/default/files/documentation/esp32_technical_reference_manual_en.pdf#63e) (page 63, Section 26.6).

Reply:
Just for good measure:

```
>>> import machine
>>> machine.mem32[0x3ff20e44]
0
>>> machine.mem32[0x3ff20e44]
0
>>> machine.mem32[0x3ff20e44]
0
>>> machine.mem32[0x3FF75144]
-2144667706
>>> machine.mem32[0x3FF75144]
-1049645602
>>> machine.mem32[0x3FF75144]
-410141976
>>> machine.mem32[0x3ff20e44]
0
>>> import os
>>> os.uname()
(sysname='esp32', nodename='esp32', release='1.13.0', version='v1.13 on 2020-09-02', machine='ESP32 module with ESP32')
```

Reply:
This bit of the code changes the esp8266 port so is correct.